ReText/tab.py

```diff
@@ -275,13 +283,19 @@ def startPendingConversion(self):
 			self.converterProcess.start_conversion(self.getActiveMarkupClass().name,
 			                                       self.fileName,
 			                                       requested_extensions,
-			                                       self.editBox.toPlainText(),
+			                                       self.cacheRemotelyHostedImages(self.editBox.toPlainText()),
```

Comment:
You are calling this on the source Markdown/reST file, not on the resulting HTML. The source file will usually not have `<img>` tags.

Reply:
That's true but they can. So I think we would have to support them.

Reply:
I mean you should parse the converted output, not the source output. See e.g. the `html` variable in `updatePreviewBox()` method.

Parsing Markdown with regular expressions is a bad idea as there are always corner cases that you will miss.

And there is also reStructuredText, Textile and other markups…

If you parse the converted HTML then there is only one language to parse.

Reply:
I see what you mean now. This should be resolved now.